nixos/modules/services/misc/mediatomb.nix

```diff
@@ -257,12 +270,12 @@ in {
   ###### implementation
 
   config = mkIf cfg.enable {
-    systemd.services.mediatomb = {
-      description = "MediaTomb media Server";
+    systemd.services."${name}"= {
+      description = "${cfg.serverName} media Server";
       after = [ "network.target" ];
       wantedBy = [ "multi-user.target" ];
-      path = [ pkgs.mediatomb ];
-      serviceConfig.ExecStart = "${pkgs.mediatomb}/bin/mediatomb -p ${toString cfg.port} ${if cfg.interface!="" then "-e ${cfg.interface}" else ""} ${if cfg.customCfg then "" else "-c ${mtConf}"} -m ${cfg.dataDir}";
+      path = [ pkg ];
```

Comment:
Is this line actually required?

Reply:
I gather it's not since the command line specifies the full path to the binary...
We'll see ;)

Reply:
As expected, it works without it ;)